db/models.py

```diff
@@ -50,3 +53,71 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders")
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets")
+    order = models.ForeignKey(Order,
+                              on_delete=models.CASCADE,
+                              related_name="tickets")
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def clean(self) -> None:
+        if not (1 <= self.row <= self.movie_session.cinema_hall.rows):
+            raise ValidationError(
+                {
+                    "row": f"row number must be in available range: "
+                    f" (1, rows): "
+                    f"(1, {self.movie_session.cinema_hall.rows})"
+                }
+            )
+
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            raise ValidationError(
+                {
+                    "seat": f"seat number must be in available range: "
+                    f" (1, seats_in_row): "
+                    f"(1, {self.movie_session.cinema_hall.seats_in_row})"
+
+                }
+            )
+
+    def save(self, *args, **kwargs) -> object:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["movie_session", "row", "seat"],
+                name="unique_movie_session_row_seat",
+            )
+        ]
+
+    def __str__(self) -> str:
+        return \
+            (f"{self.movie_session.movie.title} "
+             f"{self.movie_session.show_time.strftime("%Y-%m-%d %H:%M:%S")} "
+             f"(row: {self.row}, seat: {self.seat})")
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The custom User model inherits from `AbstractUser`, which is correct for extending the default User model. Ensure that any additional fields or methods required for your application are implemented here.